src/rules/__tests__/no-restricted-matchers.test.ts

```diff
@@ -70,7 +70,7 @@ ruleTester.run('no-restricted-matchers', rule, {
       ],
     },
     {
-      code: 'expect(a).not',
+      code: 'expect(a).not[x]()',
```

Comment:
shouldn't we keep the old test?

Reply:
The new parser doesn't consider the old test to be a valid expect call, and we've already got `expect(a).rejects` as a `valid` test which covers this change 🤷